Test/CoreSDK.Test/Shared/Extensibility/TelemetryConfigurationFactoryTest.cs

```diff
@@ -56,6 +61,34 @@ public void InstanceIsLazilyInitializedToSimplifyResettingOfGlobalStateInTests()
             Assert.NotNull(TelemetryConfigurationFactory.Instance);
         }
 
+        [TestMethod]
+        public void InstanceGetterLogsMessageToCoreEventSource()
+        {
+            using (var listener = new TestEventListener())
+            {
+                listener.EnableEvents(CoreEventSource.Log, EventLevel.Verbose);
+
+                // Invoke Getter and verify that atleast one message is logged to CoreEventSource.
+                TelemetryConfigurationFactory.Instance = null;
+                TelemetryConfigurationFactory retInstance = TelemetryConfigurationFactory.Instance;                
+                Assert.True(listener.Messages.Count() > 0, "TelemetryConfigurationFactory setter did not any message to CoreEventSource");
```

Comment:
nitpick: grammar here and below: "TelemetryConfigurationFactory setter did not send a message to CoreEventSource."